src/requirements/data/majors/math.ts

```diff
@@ -0,0 +1,396 @@
+    checker: includesWithSingleRequirement('MATH 2220', 'MATH 2240', 'MATH 1920'),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Programming Course',
+    description: 'A 3- or 4-credit computer programming course',
+    source: 'https://math.cornell.edu/major',
+    checker: includesWithSingleRequirement('CS 1110', 'CS 1112', 'CS 2110', 'CS 2112'),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Algebra',
+    description: 'Two Courses in Algebra',
+    source: 'http://pi.math.cornell.edu/files/Undergraduate/Major/worksheets/u-mathmajor-math.pdf',
+    //  TODO - We don't account for forbidden overlaps
+    checker: includesWithSingleRequirement(
+      'MATH 3320',
+      'MATH 3340',
+      'MATH 4310',
+      'MATH 4330',
+      'MATH 4340',
+      'MATH 4370',
+      'MATH 4500',
+      'MATH 4560',
+      'MATH 3360',
+      'MATH 4315'
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [2],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Analysis',
+    description: 'Two Courses in Analysis',
+    source: 'http://pi.math.cornell.edu/files/Undergraduate/Major/worksheets/u-mathmajor-math.pdf',
+    //  TODO - We don't account for forbidden overlaps
+    checker: includesWithSingleRequirement(
+      'MATH 3110',
+      'MATH 3210',
+      'MATH 3230',
+      'MATH 4130',
+      'MATH 4140',
+      'MATH 4180',
+      'MATH 4200',
+      'MATH 4210',
+      'MATH 4220',
+      'MATH 4250',
+      'MATH 4260',
+      'MATH 4280'
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [2],
+    slotNames: ['Courses'],
+  },
+  {
+    name: 'Concentration',
+    // TODO - Double Concentrations?
+    description:
+      'Students are required to complete AT LEAST one concentration from the eight concentrations available',
+    source: 'https://math.cornell.edu/major#major-requirements',
+    fulfilledBy: 'toggleable',
+    fulfillmentOptions: {
+      Mathematics: {
+        description:
+          '(i) Four 3000+ MATH courses (At least one of the four courses must be among the geometry/topology courses), ' +
+          '(ii) One course dealing with mathematical models',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['MATH 3***', 'MATH 4***', 'MATH 5***', 'MATH 6***', 'MATH 7***'],
+          [
+            'MATH 3210',
+            'MATH 4500',
+            'MATH 4520',
+            'MATH 4530',
+            'MATH 4540',
+            'MATH 4550',
+            'MATH 4560',
+          ],
+          ['MATH 3610', 'CS 2110', 'PHYS 1116', 'PHYS 2208', 'PHYS 2213', 'PHYS 2217']
+        ),
+        perSlotMinCount: [3, 1, 1],
+        slotNames: ['3000+ Math Courses', 'Geometry/Topology Course', 'Mathematical Models Course'],
+      },
+      'Applied Mathematics': {
+        description:
+          '5 courses from (i) and (ii): ' +
```

Comment:
same here: say 5 additional courses and for the other descriptions of the concentrations